src/config.h

```diff
@@ -132,12 +132,21 @@ enum {
 	SCENARIO_SELECT_MODE_DIFFICULTY,
 	SCENARIO_SELECT_MODE_ORIGIN,
 };
+typedef struct {
+	utf8string game_path;
+	utf8string game_path_slash;
+	utf8string saved_game_path;
+	utf8string scenario_path;
+	utf8string landscapes_path;
+	utf8string object_data_path;
+	utf8string tracks_path;
+	utf8string saved_game_path_2;
```

Comment:
I think this is legacy of the original game and doesn't need to be used any more. I am not sure what was different about it.